tests/conftest.py

```diff
@@ -251,6 +253,9 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "airflow_2: mark tests that works only on Airflow 2.0 / master"
     )
+    config.addinivalue_line(
+        "markers", "credential_env(name): mark tests that require credential env in CREDENTIALS_ENV"
+    )
```

Comment:
I think it would be simple to add
```
@unittest.skipIf(os.environ.get("DATAPREP_TOKEN") is None, 'Dataprep token not present')
```
WDYT?

Reply:
It's simpler. Indeed. 
Do you think that my solution in over engineering? I like that solution because of explicit message for the user and it's reusable.

Reply:
I think this is indeed an over-engineering. To run this test I have to create a file that requires much more work than exporting an env variable. It also assumes that the test is run from breeze environment or that I set up another env variable that points to other directory...

Reply:
Done.